packages/react/src/primitives/Select/Select.tsx

```diff
@@ -81,6 +81,7 @@ const SelectPrimitive: Primitive<SelectProps, 'select'> = (
             classNameModifier(ComponentClassName.SelectIcon, size)
           )}
           color={iconColor}
+          aria-hidden="true"
```

Comment:
Can we do a quick test with a screen reader to make sure this prevents this from being read out as "image"?

Reply:
Verified